keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -759,26 +761,13 @@ This can inform certain test coverage improvements that we want to do before
 extending the production code to implement this enhancement.
 -->
 
-- `<package>`: `<date>` - `<test coverage>`
+- `pkg/kubelet/images`: `2-10-2024` - `83.8`
+- `pkg/kubelet/kuberuntime`: `2-10-2024` - `66.6`
 
 ##### Integration tests
 
-<!--
-Integration tests are contained in k8s.io/kubernetes/test/integration.
-Integration tests allow control of the configuration parameters used to start the binaries under test.
-This is different from e2e tests which do not allow configuration of parameters.
-Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
--->
-
-<!--
-This question should be filled when targeting a release.
-For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
-
-For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
-https://storage.googleapis.com/k8s-triage/index.html
--->
-
-- <test>: <link to test coverage>
+Integration tests for the kubelet are not existing yet and will be covered by
+extending the e2e_node, e2e and unit tests.
```

Comment:
critest maybe?

Reply:
that'd test the CRI, which i'd say is out of scope for this

Reply:
Are you scoping the tests to a mock CRI?

Reply:
not following the this isn't CRI when the CRI server is the volume manager for the OCI image volumes.. or maybe I've missed something?